src/main/java/mekanism/common/block/BlockEnergyCube.java

```diff
@@ -284,35 +255,25 @@ public EnumBlockRenderType getRenderType(IBlockState state)
 		return EnumBlockRenderType.MODEL;
 	}
 
-	@Override
-    public ItemStack getPickBlock(IBlockState state, RayTraceResult target, World world, BlockPos pos, EntityPlayer player)
-	{
+	private ItemStack getDropItem(IBlockAccess world, BlockPos pos) {
 		TileEntityEnergyCube tileEntity = (TileEntityEnergyCube)world.getTileEntity(pos);
+
 		ItemStack itemStack = new ItemStack(MekanismBlocks.EnergyCube);
-		
-		if(!itemStack.hasTagCompound())
+		if (tileEntity == null)
 		{
-			itemStack.setTagCompound(new NBTTagCompound());
+			throw new NullPointerException();
```

Comment:
You must give it a message if you insist on throwing. Though I don't see much point.

Reply:
Removed as you asked.